README.md

```diff
@@ -1,21 +1,26 @@
-# SendEmail
+<div align="center">
 
-[![Build Status](https://travis-ci.org/dwyl/sendemail.svg?branch=tidyup)](https://travis-ci.org/dwyl/sendemail)
-[![codecov.io](https://codecov.io/github/dwyl/sendemail/coverage.svg?branch=master)](https://codecov.io/github/dwyl/sendemail?branch=master)
-[![Code Climate](https://codeclimate.com/github/dwyl/sendemail/badges/gpa.svg)](https://codeclimate.com/github/dwyl/sendemail)
-[![Dependency Status](https://david-dm.org/dwyl/sendemail.svg)](https://david-dm.org/dwyl/sendemail)
-[![devDependency Status](https://david-dm.org/dwyl/sendemail/dev-status.svg)](https://david-dm.org/dwyl/sendemail#info=devDependencies)
-[![JavaScript Style Guide: Good Parts](https://img.shields.io/badge/code%20style-goodparts-brightgreen.svg?style=flat)](https://github.com/dwyl/goodparts "JavaScript The Good Parts")
+# `sendemail` 💌
 
-[![Node.js Version](https://img.shields.io/node/v/sendemail.svg?style=flat "Node.js 0.12 and 4.x latest both supported")](http://nodejs.org/download/)
-[![npm](https://img.shields.io/npm/v/sendemail.svg)](https://www.npmjs.com/package/sendemail)
-[![Join the chat at https://gitter.im/dwyl/chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/dwyl/chat/?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Effortlessly send email from any Node.js App
+using Amazon's Simple Email Service (SES). <br />
+e.g: ***welcome***, **newsletter**, **verification**,
+**password reset** and **notification** emails
 
-*Send* ***welcome, verification, password reset, update and reminder emails***
-from *any* node.js app.
 
+[![Build Status](https://img.shields.io/travis/dwyl/sendemail/master.svg?style=flat-square)](https://travis-ci.org/dwyl/sendemail)
+[![codecov.io](https://img.shields.io/codecov/c/github/dwyl/sendemail/master.svg?style=flat-square)](http://codecov.io/github/dwyl/sendemail?branch=master)
+[![Code Climate maintainability](https://img.shields.io/codeclimate/maintainability/dwyl/sendemail?color=brightgreen&style=flat-square)](https://codeclimate.com/github/dwyl/sendemail)
+[![dependencies Status](https://david-dm.org/dwyl/sendemail/status.svg?style=flat-square)](https://david-dm.org/dwyl/sendemail)
+[![devDependencies Status](https://david-dm.org/dwyl/sendemail/dev-status.svg?style=flat-square)](https://david-dm.org/dwyl/sendemail?type=dev)
+[![HitCount](http://hits.dwyl.com/dwyl/sendemail.svg)](http://hits.dwyl.com/dwyl/sendemail)
```

Comment:
We already know that currently the `hits` subdomain is down https://github.com/dwyl/hits/issues/93 so this badge will show as a broken image, but the link is solid otherwise. Leaving this in as it will function once that issue is resolved.